### PR TITLE
Support password flag

### PR DESCRIPTION
**Pull request by appleboy**
Also fix #21 

* Fix load ENV from the config file.
* Support password flag.
* update documents.

cc @tboerger @bradrydzewski @Bugagazavr

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,8 +44,9 @@ docker run --rm \
   -e PLUGIN_HOST=foo.com \
   -e PLUGIN_USER=root \
   -e PLUGIN_KEY="$(cat ${HOME}/.ssh/id_rsa)" \
+  -e PLUGIN_PASSWORD=xxxxx \
```
Comment (tboerger) [inline README.md]: You should set the key or the password but not both

File: main.go [hunk 2]
```diff
@@ -56,8 +62,9 @@ func main() {
 			EnvVar: "PLUGIN_SCRIPT,SSH_SCRIPT",
 		},
 		cli.StringFlag{
-			Name:  "env-file",
-			Usage: "source env file",
+			Name:   "env-file",
+			Usage:  "source env file",
+			EnvVar: "ENV_FILE",
```
Comment (tboerger) [inline main.go]: Nope, we have not done that intentionally.

File: README.md (hunk 1 above)
Comment (appleboy) [inline README.md]: OK. Removed

File: main.go (hunk 2 above)
Comment (appleboy) [inline main.go]: Removed.

**Comment by appleboy** [comment]
@tboerger Do you have any concerns of this PR?

**Comment by tboerger** [comment]
LGTM

**Comment by baskuis** [comment]
Hi - we ran into an issue with this build. Not a huge one, but when we set a SSH key but no PASSWORD - we saw the following error message: 

Error: Failed to parse private key. ssh: no key found

This started happening after this was merged.

The work around for us - was simply to set a (bad) pw.

**Comment by appleboy** [comment]
@baskuis I will take it.

**Comment by appleboy** [comment]
@baskuis What is your drone version? latest?

**Pull request merged**